src/script/components/ConnectRequests/index.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FC, useEffect, useRef} from 'react';
+import {container} from 'tsyringe';
+
+import Avatar, {AVATAR_SIZE} from 'Components/Avatar';
+import ClassifiedBar from 'Components/input/ClassifiedBar';
+
+import {registerReactComponent, useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+
+import {TeamState} from '../../team/TeamState';
+import {UserState} from '../../user/UserState';
+import {User} from '../../entity/User';
+import {ActionsViewModel} from '../../view_model/ActionsViewModel';
+
+interface ConnectRequestsProps {
+  readonly actionsViewModel: ActionsViewModel;
+  readonly userState: UserState;
+  readonly teamState: TeamState;
+}
+
+const ConnectRequests: FC<ConnectRequestsProps> = ({
```

Comment:
I would ask for removal of FC here as well. Its just a not needed import / functionality since (props: Type) does the same